### ISSUE TITLE
Train

### ISSUE THREAD

**Issue by Pantonius** [Pantonius/mini-nn-verifier#4]
`train --output-dir <dir> <dataset-name> <train_inputs.bin> <train_labels.bin>`

For `train`, write `eval_batch_size: 〈N〉` as the first line of standard output, then print
the absolute path of each checkpoint file inside `--output-dir`, one path per line. The
value `N` is the batch size the test runner uses when evaluating your checkpoints with `eval`.

- `〈dataset-name〉` argument is a string identifier (such as `mnist` or `mnist_mlp`) that your implementation uses to select the model architecture and any training hyperparameters. 
- `〈train_inputs.bin〉` file contains a float64 input array of shape (N, d) and the
- `〈train_labels.bin〉` file contains a float64 one-hot label array of shape (N, C), both in the .bin format described below.
 
For the datasets used in the test suite, d = 784 (flattened 28×28 grayscale images with values in [0, 1]) and C ∈{5, 10} depending on the dataset. Each checkpoint your container emits must be a valid .mininn file; the test runner evaluates every checkpoint with eval on batches of shape (eval_batch_size, d) and expects class scores of shape (eval_batch_size, C) as output. Predictions are obtained by taking the argmax over the last axis and compared to the argmax of the one-hot labels.
